data/bL34cHig0.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Habeeb Kareem",
+  "bio": "Aspiring Pentester | IT graduate | Open-Source Enthusiast"
```

Comment:
Yes, you should add a `,` in the end of line 3.

Reply:
oh yeah, thanks for pointing that out for me.